portal/db.py

```diff
@@ -70,7 +73,7 @@ def mock_db():
         with get_db() as con:
             with con.cursor() as cur:
                 cur.execute(f.read())
-
+    mock_csv()  
```

Comment:
Potential nitpick: should the CSV be treated as mock data?

In my mind, the purpose of this `mock_db` function is to populate the test database with all manner of data for the tests to use, good and bad, complete and incomplete, whatever you need for tests. But the purpose of the CSV import feature is to register real users into the database to use the application, right? Why combine them?